packages/compute-baseline/src/browser-compat-data/supportStatements.test.ts

```diff
@@ -186,5 +195,147 @@ describe("statements", function () {
+        assert.equal(unranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(unranged.supportedIn(cr.current()).supported, true);
+        assert.equal(
+          unranged.supportedIn(cr.releases.at(-1) as any).supported,
+          true,
+        );
+
+        assert.equal(ranged.supportedIn(cr.version("99")).supported, null);
+        assert.equal(ranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(ranged.supportedIn(cr.current()).supported, true);
+        assert.equal(
+          ranged.supportedIn(cr.releases.at(-1) as any).supported,
+          true,
+        );
+      });
+
+      it("returns supported when release is on after version_added and before version_removed", function () {
+        const cr = browser("chrome");
+        const unranged = new RealSupportStatement(
+          { version_added: "100", version_removed: "125" },
+          cr,
+        );
+        const ranged = new RealSupportStatement(
+          { version_added: "≤100", version_removed: "125" },
+          cr,
+        );
+
+        assert.equal(unranged.supportedIn(cr.version("99")).supported, false);
+        assert.equal(unranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("124")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("125")).supported, false);
+
+        assert.equal(ranged.supportedIn(cr.version("99")).supported, null);
+        assert.equal(ranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("124")).supported, true);
+      });
+
+      it("returns unknown support when release is before ranged version_added", function () {
+        const cr = browser("chrome");
+        const rangedOpen = new RealSupportStatement(
+          { version_added: "≤100" },
+          cr,
+        );
+        const rangedClosed = new RealSupportStatement(
+          { version_added: "≤100", version_removed: "125" },
+          cr,
+        );
+
+        assert.equal(rangedOpen.supportedIn(cr.version("99")).supported, null);
+        assert.equal(
+          rangedClosed.supportedIn(cr.version("99")).supported,
+          null,
+        );
+      });
+
+      it("returns unknown support when release is after version_added and before ranged version_removed", function () {
+        const cr = browser("chrome");
+        const rangedEnd = new RealSupportStatement(
+          { version_added: "100", version_removed: "≤125" },
+          cr,
+        );
+
+        assert.equal(rangedEnd.supportedIn(cr.version("100")).supported, true);
+        assert.equal(rangedEnd.supportedIn(cr.version("124")).supported, null);
```

Comment:
Can you also test 101 here to make it clear that there's just a single release we're sure about?

Reply:
Done with 4e0110b52d7ae60a9ced2e0c19f54ea58b8fa0cb.